content/07.pathogenesis.md

```diff
@@ -211,43 +212,41 @@ These analyses indicate that most pediatric cases of COVID-19 are not severe.
-In several preliminary assays and an analysis of previous microarray data, ACE2 expression was reported to be significantly upregulated following infection of human embryonic kidney cells and human airway epithelial cells [@doi:10.1101/2020.02.24.963348].
+For example, previous studies suggest that infection by coronaviruses, such as SARS-CoV-1 and MERS-CoV, as well as other viruses, is associated with the upregulation of ACE2.
+In several preliminary assays and an analysis of microarray data, ACE2 expression was reported to be significantly upregulated following infection of human embryonic kidney cells and human airway epithelial cells [@doi:10.1101/2020.02.24.963348].
 This study also reported that direct stimulation with inflammatory cytokines such as type I interferons (e.g., IFN&beta;) resulted in the upregulation of ACE2 in human bronchial epithelial cells, with treated groups showing four-fold higher ACE2 expression than control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
-Whether SARS-CoV-2 facilitates the positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.
-A systems-biology approach allows for analyses such as these to identify possible phenotypic and endophenotypic responses to SARS-CoV-2 infection and to develop new hypotheses about how pathogenesis proceeds.
+While it is still unclear whether SARS-CoV-2 facilitates the positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.
 
 #### Transcriptomics
 
 In addition to the study described above, two other studies have profiled expression following SARS-CoV-2 infection using human cell lines.
 The first study [@doi:10.1016/j.cell.2020.04.026] compared transcriptional responses to SARS-CoV-2 and to other respiratory viruses, including MERS-CoV, SARS-CoV, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV).
-The authors analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
-As the viral receptor ACE2 has low expression in A549 cells, they supplemented the A549 cells with adenovirus (AdV)-based vectors expressing either mCherry (a fluorescent protein used as a control) or ACE2 (A549-ACE2).
+They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
```

Comment:
"They" now refers to "The first study".  That doesn't read naturally to me.

Reply:
```suggestion
The responses of three human cell lines were analyzed: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
```
Passive voice to the rescue!